src/taker/api.rs

```diff
@@ -1953,7 +1951,7 @@ impl Taker {
             maker_count,
             config.connection_type,
         )?;
-        let offers = fetch_offer_from_makers(addresses_from_dns, config);
+        let offers = fetch_offer_from_makers(addresses_from_dns, &config);
```

Comment:
```suggestion
        let offers = fetch_offer_from_makers(addresses_from_dns, &self.config);
```

Reply:
We should not do that. We just need to pass the reference. putting it in a variable and then passing the reference is suboptimal.